components/microsoft_onedrive/actions/get-table/get-table.mjs

```diff
@@ -0,0 +1,92 @@
+import axios from "@pipedream/platform";

```

Comment:
Correct import:

```suggestion
import { axios } from "@pipedream/platform";
```


Also, please run the following so that `package.json` and `pnpm-lock.yaml` are updated.
1. `npm install @pipedream/platform` on `/components/microsoft_onedrive`
2. `npx pnpm install -r` on project root `/`